tests/cephfs/cephfs_mirroring/test_cephfs_mirroring_configure_cephfs_mirroring.py

```diff
@@ -0,0 +1,333 @@
+                "size": "5368709120",
+            },
+        ]
+        for subvolume in subvolume_list:
+            fs_util_ceph1.create_subvolume(source_clients[0], **subvolume)
+
+        mounting_dir = "".join(
+            random.choice(string.ascii_lowercase + string.digits)
+            for _ in list(range(10))
+        )
+        log.info("Mount 1 subvolume on kernel and 1 subvloume on Fuse → Client1")
+        kernel_mounting_dir_1 = f"/mnt/cephfs_kernel{mounting_dir}_1/"
+        mon_node_ips = fs_util_ceph1.get_mon_node_ips()
+        log.info("Get the path of subvolume1 on  filesystem")
+        subvol_path1, rc = source_clients[0].exec_command(
+            sudo=True,
+            cmd=f"ceph fs subvolume getpath {source_fs} subvol_1 subvolgroup_1",
+        )
+        index = subvol_path1.find("subvol_1/")
+        if index != -1:
+            subvol1_path = subvol_path1[: index + len("subvol_1/")]
+        else:
+            subvol1_path = subvol_path1
+        log.info(subvol1_path)
+
+        fs_util_ceph1.kernel_mount(
+            [source_clients[0]],
+            kernel_mounting_dir_1,
+            ",".join(mon_node_ips),
+        )
+        log.info("Get the path of subvolume2 on filesystem")
+        fuse_mounting_dir_1 = f"/mnt/cephfs_fuse{mounting_dir}_1/"
+        subvol_path2, rc = source_clients[0].exec_command(
+            sudo=True,
+            cmd=f"ceph fs subvolume getpath {source_fs} subvol_2 subvolgroup_1",
+        )
+        index = subvol_path2.find("subvol_2/")
+        if index != -1:
+            subvol2_path = subvol_path2[: index + len("subvol_2/")]
+        else:
+            subvol2_path = subvol_path2
+        log.info(subvol2_path)
+        fs_util_ceph1.fuse_mount(
+            [source_clients[0]],
+            fuse_mounting_dir_1,
+        )
+
+        log.info("Add subvolumes for mirroring to remote location")
+        fs_mirroring_utils.add_path_for_mirroring(
+            source_clients[0], source_fs, subvol1_path
+        )
+        fs_mirroring_utils.add_path_for_mirroring(
+            source_clients[0], source_fs, subvol2_path
+        )
+
+        log.info("Add files into the path and create snapshot on each path")
+        snap1 = "snap_k1"
+        snap2 = "snap_f1"
+        file_name1 = "hello_kernel"
+        file_name2 = "hello_fuse"
+        source_clients[0].exec_command(
+            sudo=True, cmd=f"touch {kernel_mounting_dir_1}{subvol1_path}hello_kernel"
+        )
+        source_clients[0].exec_command(
+            sudo=True, cmd=f"touch {fuse_mounting_dir_1}{subvol2_path}hello_fuse"
+        )
+        source_clients[0].exec_command(
+            sudo=True, cmd=f"mkdir {kernel_mounting_dir_1}{subvol1_path}.snap/{snap1}"
+        )
+        source_clients[0].exec_command(
+            sudo=True, cmd=f"mkdir {fuse_mounting_dir_1}{subvol2_path}.snap/{snap2}"
+        )
+
+        log.info("Wait for the snapshot to synchronize")
+        time.sleep(60)
```

Comment:
instead of waiting for 60secs, can we retry after 10secs?

Reply:
Sure, I ll change it.